src/main/scala/mesosphere/marathon/core/storage/repository/impl/legacy/GroupEntityRepository.scala

```diff
@@ -0,0 +1,53 @@
+package mesosphere.marathon.core.storage.repository.impl.legacy
```

Comment:
Please move this class into the store module. The repository module should not have any knowledge about specific types.